llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -2658,7 +2659,9 @@ void InnerLoopVectorizer::createVectorLoopSkeleton(StringRef Prefix) {
   LoopScalarPreHeader =
       SplitBlock(LoopVectorPreHeader, LoopVectorPreHeader->getTerminator(), DT,
                  LI, nullptr, Twine(Prefix) + "scalar.ph");
-  replaceVPBBWithIRVPBB(Plan.getScalarPreheader(), LoopScalarPreHeader);
+  // NOTE: The Plan's scalar preheader isn't replaced with a VPIRBasicBlock
+  // wrapping LoopScalarPreHeader here at the moment, because the Plan's scalar
+  // preheader may be unreachable at this point.
```

Comment:
Is it worth helping the reader by pointing to the place where we do replace it, i.e. `createVectorizedLoopSkeleton`?

Reply:
Done, thanks!